projects/plugins/jetpack/extensions/blocks/premium-content/editor.scss

```diff
@@ -100,7 +62,7 @@
 
 /* Hide Stripe nudge from Jetpack payments block */
 /* https://github.com/Automattic/wp-calypso/issues/44332 */
-.wp-block-premium-content-container .jetpack-block-nudge {
+.wp-block-premium-content-container .premium-content-wrapper .jetpack-block-nudge {
```

Comment:
We need a little more specificity than before, we now want to show the main Stripe Nudge but hide anny inner recurring payments Stripe Nudge.